packages/cinterop/src/nativeDarwin/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt

```diff
@@ -2464,6 +2464,30 @@ actual object RealmInterop {
         }
     }
 
+    actual fun realm_app_call_reset_password_function(
+        app: RealmAppPointer,
+        email: String,
+        newPassword: String,
+        serializedEjsonPayload: String,
+        callback: AppCallback<Unit>
+    ) {
+        memScoped {
+            checkedBooleanResult(
+                realm_wrapper.realm_app_email_password_provider_client_call_reset_password_function(
+                    app.cptr(),
+                    email,
+                    newPassword.toRString(this),
+                    serializedEjsonPayload,
+                    staticCFunction { userData, error ->
+                        handleAppCallback(userData, error) { /* No-op, returns Unit */ }
```

Comment:
not completely sure if this is correct, just going off of similar functions

Reply:
This looks fine 👍